### PR TITLE
style: add updated fonts

### PR DESCRIPTION
**Pull request by kayleegeorge**
Add font file based on updated theme

### CODE REVIEW

File: src/theme/fonts.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+export const fontFamily = {
```
Comment (vm) [inline src/theme/fonts.ts:1]: don't these need to get added to to `useTheme`?

Comment (kayleegeorge) [reply src/theme/fonts.ts:1]: oh wait ya, I need to do the text style differently -- ignore this PR for now :)

File: src/theme/index.tsx [hunk 2]
```diff
@@ -258,6 +268,45 @@ export const ThemedText = {
   DeprecatedError({ error, ...props }: { error: boolean } & TextProps) {
     return <TextWrapper fontWeight={500} color={error ? 'deprecated_red1' : 'deprecated_text2'} {...props} />
   },
+  HeadlineLarge({ error, ...props }: { error: boolean } & TextProps) {
```
Comment (vm) [inline src/theme/index.tsx:271]: i think we probably want to stop using these `TextWrapper` components. probably can avoid for now? maybe just add the new fontFamily stuff into theme and that's enough

Comment (kayleegeorge) [reply src/theme/index.tsx:271]: Ya I was curious about this too, but I think design wanted to capture the font size / line height too

**Comment by kayleegeorge** [comment]
Deciding not to use this TextWrapper model for updating fonts so not needed

**Pull request closed without merging**